<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1030</width>
    <height>709</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Fdog-Kit-Zookeeper</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow,QWidget#centralWidget {
background-color: rgb(67, 67, 67);
border-radius:10px;
}</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="styleSheet">
    <string notr="true">#centralWidget {
border-image: url(:/lib/back1.png);
border-radius:10px;
}</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QWidget" name="widget_title" native="true">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>30</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>30</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgba(30, 45, 54, 0);
color: rgb(255, 255, 255);
</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QToolButton" name="toolButton_min_2">
         <property name="minimumSize">
          <size>
           <width>40</width>
           <height>29</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>40</width>
           <height>29</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>10</pointsize>
           <stylestrategy>PreferAntialias</stylestrategy>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QToolButton {
	color: rgb(255, 255, 255);
	
	background-color: rgba(94, 255, 210, 0);
	border: none;
}

QToolButton::menu-indicator { 
	image: None;
}

 
QToolButton:hover {
	color: rgb(255, 255, 255);

	
	background-color: rgb(23, 35, 42);
	border: none;
}</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="lib.qrc">
           <normaloff>:/lib/icon-minus3.png</normaloff>:/lib/icon-minus3.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>13</width>
           <height>13</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="toolButton_max_2">
         <property name="minimumSize">
          <size>
           <width>40</width>
           <height>29</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>40</width>
           <height>29</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>10</pointsize>
           <stylestrategy>PreferAntialias</stylestrategy>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QToolButton {
	color: rgb(255, 255, 255);
	
	background-color: rgba(94, 255, 210, 0);
	border: none;
}

QToolButton::menu-indicator { 
	image: None;
}

 
QToolButton:hover {
	color: rgb(255, 255, 255);
	background-color: rgb(23, 35, 42);
	border: none;
}</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="lib.qrc">
           <normaloff>:/lib/Icon_max4.png</normaloff>:/lib/Icon_max4.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>11</width>
           <height>11</height>
          </size>
         </property>
         <property name="checkable">
          <bool>false</bool>
         </property>
         <property name="autoRepeat">
          <bool>false</bool>
         </property>
         <property name="autoExclusive">
          <bool>false</bool>
         </property>
         <property name="autoRaise">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="toolButton_close_2">
         <property name="minimumSize">
          <size>
           <width>40</width>
           <height>29</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>40</width>
           <height>29</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>10</pointsize>
           <stylestrategy>PreferAntialias</stylestrategy>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QToolButton {
	color: rgb(255, 255, 255);
	border-top-right-radius: 9px;
	background-color: rgba(94, 255, 210, 0);
	border: none;
}

QToolButton::menu-indicator { 
	image: None;
}

 
QToolButton:hover {
	color: rgb(255, 255, 255);
	
	background-color: rgb(200, 0, 0);
	border: none;
}</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="lib.qrc">
           <normaloff>:/lib/icon-close8.png</normaloff>:/lib/icon-close8.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>11</width>
           <height>11</height>
          </size>
         </property>
         <property name="checkable">
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="widget_body" native="true">
      <layout class="QVBoxLayout" name="verticalLayout_2"/>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="lib.qrc"/>
 </resources>
 <connections/>
</ui>
